### ISSUE TITLE
Scaling issue 

### ISSUE THREAD

**Issue by LolaaPe** [jo-mueller/Slice2Volume#26]
Hi Johannes,
When running S2V with the attacched parameters, the Output Stack doesn't fit the atlas I registered my histological images to (image attached). The output stack seemes to be much smaller than the atlas.
Thanks for your help!

![S2V scaling issue](https://github.com/jo-mueller/Slice2Volume/assets/124932322/d6d37bb9-33e3-4d46-8356-610b27193895)

![S2V parameters](https://github.com/jo-mueller/Slice2Volume/assets/124932322/74db0c0b-41ff-417c-b3cf-baf1c5e9f649)

**Comment by jo-mueller** [comment]
Hi @LolaaPe ,

thanks for reporting! I encountered this error earlier and thought I had it fixed for *most* cases, but apparently that's not the case. Two questoins:
- When you run S2V on histological data, you should be presented with a dialogue - something like `Image is much larger than target - proceed with downsampled image?`. This you have to confirm, otherwise it will not work.
- Secondly, does this occur for all slices or just for a few ones?

Things you could try:
- Try increasing the `histo outline smoothing degree` and the `Volume outline smoothing degree` (in the S2V publication I used values up to 10 or 12).
- If you look into the `elastix_parameters.txt`, there are these three lines:

```
(NumberOfResolutions 5)
(FixedImagePyramidSchedule 64 64 16 16 8 8 2 2 1 1)
(MovingImagePyramidSchedule 16 16 16 16 8 8 2 2 1 1)
```

You could try increasing the `NumberOfResolutions` parameter to 6, in which case you would have to change the two other lines into something like this:

```
(FixedImagePyramidSchedule 82 82 64 64 16 16 8 8 2 2 1 1)
(MovingImagePyramidSchedule 82 82 16 16 16 16 8 8 2 2 1 1)
```

Basically, this blurs both the fixed and the target image (the atlas image and the histological image to be registered) A LOT so that they overlap better - the registration is then able to do the rest. However, if they do not overlap well in the beginning, elastix doesn't know how to change the transformation in order to make the registration better - hope this makes sense.

**Comment by LolaaPe** [comment]
Solution: https://forum.image.sc/t/splitting-large-multi-channel-czi-to-single-channel-tif/85291/2

**Comment by jo-mueller** [comment]
Thanks for linking the solution here @LolaaPe ! (And sorry for not getting back to your last comment, pretty swamped right now 🙈 )
